src/helpers.ts

```diff
@@ -16,7 +16,8 @@ export const baseTypeConversionMap = new Map<string, string>([
     ["sequence", "Array"],
     ["record", "Record"],
     ["FrozenArray", "ReadonlyArray"],
-    ["EventHandler", "EventHandler"]
+    ["EventHandler", "EventHandler"],
+    ["undefined", "void"]
```

Comment:
BTW, #920 did this in another way so we don't need this anymore. Sorry, but still was a good work!